org.uqbar.project.wollok.lib/src/wollok.wlk

```diff
@@ -512,6 +520,16 @@ package lang {
 		
 		override method internalToSmartString(alreadyShown) = start.toString() + ".." + end.toString()
 	}
+	
+	/**
+	 * @author jfernandes
+	 * @since 1.3
+	 * @noInstantiate
+	 */
+	class Closure {
+		method apply(parameters...) native
```

Comment:
eventually we will have more methods here like "curry" and maybe even some almost reflectives like "getParameterNames()" ?